quadcore/manager/data.py

```diff
@@ -35,7 +36,7 @@ def set_entity(cls, entity):
         new_key = "entity:" + str(entity_count + 1)
 
         # Set key to Entity
-        entity.entity_id = new_key
+        entity.entity_key = new_key
```

Comment:
this is the mess we'll be creating with more id space.

see if we can just use entity_id.